projects/core/src/configurator/commons/store/actions/configurator.action.ts

```diff
@@ -314,6 +322,47 @@ export class SetNextOwnerCartEntry extends StateUtils.EntitySuccessAction {
     super(CONFIGURATION_DATA, payload.configuration.owner.key);
   }
 }
+
+export class SetCurrentGroup extends StateUtils.EntitySuccessAction {
+  readonly type = SET_CURRENT_GROUP;
+
+  constructor(entityKey: string | string[], public payload: any) {
```

Comment:
Action constructors should only have one argument, please check other config actions (extensibility, this was a feedback that Krzysztof already gave, I fixed this for existing cfg actions)

Reply:
Done